end_to_end_tests/run-e2e-tests.sh

```diff
@@ -7,4 +8,14 @@ EXITVAL=$?
 #Testarchiver data storing can be added here,
 #Important to exit with the right exit value from the test execution,
 #not with the exitvalue from data storing. 
-exit $EXITVAL
\ No newline at end of file
+
+
+echo "---------------------------------------------"
+echo " Archiving reports from ./logs -directory"
+echo "---------------------------------------------"
+find ./logs -name \*.xml -type f -print0 | xargs -0 -n1 testarchiver --dbengine postgresql --database $DATABASE --host $HOST \
```

Comment:
Not 100% sure if these env variables are usable in the CI environment since the ENV_FILE is kind of used for this on the docker side.

Reply:
This command is run inside docker, with the entrypoint call of end_to_end_tests/resources/Dockerfile